target/linux/ramips/image/mt7621.mk

```diff
@@ -232,6 +232,29 @@ define Device/dlink_dir-8xx-a1
 	check-size
 endef
 
+define Device/dlink_dir-xx60-a1
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 40960k
+  UBINIZE_OPTS := -E 5
+  DEVICE_VENDOR := D-Link
+  DEVICE_PACKAGES := kmod-mt7615e kmod-mt7615-firmware \
+	kmod-usb3 kmod-usb-ledtrig-usbport wpad-basic
+  KERNEL := $$(KERNEL) | uimage-padhdr 96
```

Comment:
Does this really need to be applied to the KERNEL here?

Reply:
As opposed to where?

Reply:
Well, for dir-8xx it is applied to everything except the kernel ...

Reply:
If I understood correctly it's equivalent given that uimage-padhdr is applying to the beginning of whatever is there so far.

Reply:
you could change the dir-8xx if you like but both work 
  KERNEL := $$(KERNEL) | ubootpad96
  IMAGES += factory.bin
  IMAGE/sysupgrade.bin := append-kernel | append-rootfs |\
	 append-metadata | check-size
  IMAGE/factory.bin := append-kernel | append-rootfs | check-size